backend/src/main/resources/application.properties

```diff
@@ -1 +1,11 @@
+# DATABASE CONFIGURATION
+spring.datasource.url=jdbc:mysql://localhost:3306/parcel_management?serverTimezone=Europe/Warsaw&useSSL=False
+spring.datasource.username=webapps
+spring.datasource.password=busters
+
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+spring.jpa.database=mysql
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
+
 
```

Comment:
please add server.servlet.context-path="/packagesservice"  as in https://theflyingkeyboard.atlassian.net/browse/PAC-28

Reply:
Done.